src/content/peerconnection/audio/index.html

```diff
@@ -32,50 +32,66 @@
 
 <body>
 
-  <div id="container">
```

Comment:
Please indent this again, it belongs to \<body\>. The HTML linter is not strict enough or is not running for these files it seems.